src/style/shared.css

```
@@ -0,0 +1,218 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+header {
+  position: sticky;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 200px;
+  top: 0;
+  background-color: var(--Linkbrary-bg, #f0f6ff);
+}
+
+.headerLogo .logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 133px;
+  height: 24px;
+  cursor: pointer;
+}
+
+.headerBtn button {
+  display: flex;
+  width: 128px;
+  padding: 16px 20px;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  border: 0px;
+  border-radius: 8px;
+  background: var(--gra-purpleblue-to-skyblue, linear-gradient(91deg, #6d6afe 0.12%, #6ae3fe 101.84%));
+  color: var(--Grey-Light, #f5f5f5);
+  font-family: Pretendard;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+  cursor: pointer;
+}
+
+section {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  gap: 8px;
+  background: var(--Linkbrary-bg, #f0f6ff);
+}
+
+.title {
```

이 부분도 많이 사용하는 클래스명이다보니 셀렉터 범위를 좁히는게 좋을 것 같아요! 
```suggestion
.sectionShared .title {
```

---

수정했습니다